safe-global/safe-wallet-web
# PoC: Counterfactual safe deployment

WIP

## What it solves

This adds support for counterfactual safe deployments, limited to 1/1 setups and limited to only one undeployed safe at a time.

## How this PR fixes it

- Adjust load hooks to return an empty fallback for undeployed safes instead of fetching the CGW
- Store undeployed safe in local storage

## How to test it

1. Open safe creation
2. Create a 1/1 safe
3. Observe no gas estimation on the review screen
4. Press Create
5. Observe being immediately navigated to the Dashboard
6. Observe no errors
7. Try to create another 1/1 safe
8. Observe it has the same address

## ToDo

- [ ] Add first transaction flow

## Screenshots

## Checklist
